server/models/Meeting.js

```diff
@@ -0,0 +1,16 @@
+const mongoose = require("mongoose");
+
+const meetingSchema = new mongoose.Schema({
+  user_id: {
```

Comment:
Should be `userId`

Reply:
Yes, this is my bad - I forgot about this specific feedback. I have made appropriate changes to it.